polus-k-means-clustering-plugin/src/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pandas==0.25.3
+scikit-learn==0.22.2
+numpy==1.16.4
```

Comment:
Looks like you're using an older version of numpy. Is there a reason for this? I think the base bfio container uses uses 1.18.x